tutorials/plugins/editor/making_plugins.rst

```diff
@@ -469,15 +457,62 @@ Use the following code to register a singleton from an editor plugin:
         // Replace this value with a PascalCase autoload name.
         private const string AutoloadName = "SomeAutoload";
 
-        public override void _EnterTree()
+        public override void _EnablePlugin()
         {
             // The autoload can be a scene or script file.
             AddAutoloadSingleton(AutoloadName, "res://addons/MyAddon/SomeAutoload.tscn");
         }
 
-        public override void _ExitTree()
+        public override void _DisablePlugin()
         {
             RemoveAutoloadSingleton(AutoloadName);
         }
     }
     #endif
+
+Using sub-plugins
+~~~~~~~~~~~~~~~~~
+
+Often a plugin adds multiple things, for example a custom node and a panel.
+In those cases it might be easier to have an own plugin script for each of those features.
+Sub-plugin can be used for this.
```

Comment:
```suggestion
Sub-pluginz can be used for this.
```

Reply:
Do you mean `Sub-plugins`?

Reply:
Yes my bad

Reply:
Well with English I'm never sure about anything 😅

Reply:
English is massively confusing and just three or four languages in a trenchcoat 🙃